### PR TITLE
Fix #5929, Live caches vanish when disabling updates

### LINKED ISSUE #5929: Live caches vanish when disabling updates
When you open the (new) live map, wait for caches to load and then switch to offline, the downloaded caches are removed from the map.
They should stay, obviously.

### PR DESCRIPTION
**Pull request by rsudev**
Implemented the same semantics as previously for the live map

### CODE REVIEW

File: main/src/cgeo/geocaching/maps/mapsforge/v6/NewMap.java [hunk 1]
```diff
@@ -343,9 +344,12 @@ public boolean onOptionsItemSelected(final MenuItem item) {
                 return true;
             case R.id.menu_map_live:
                 mapOptions.isLiveEnabled = !mapOptions.isLiveEnabled;
+                if (mapOptions.isLiveEnabled) mapOptions.isStoredEnabled = true;
```
Comment (pstorch) [inline main/src/cgeo/geocaching/maps/mapsforge/v6/NewMap.java]: Do we have a checkstyle rule that braces are needed? I'm not sure at the moment.

Comment (UniQP) [inline main/src/cgeo/geocaching/maps/mapsforge/v6/NewMap.java]: Yes: `main/src/cgeo/geocaching/maps/mapsforge/v6/NewMap.java:347: 'if' construct must use '{}'s.`

File: main/src/cgeo/geocaching/maps/mapsforge/v6/caches/CachesBundle.java [hunk 2]
```diff
@@ -123,6 +123,20 @@ public void handleLiveLayers(final boolean enable) {
         }
     }
 
+    /**
+     * Enables the stored cache layer. No disabling again!
+     * @param enable
+     *          true - enable stored layer, false - leave untouched
+     */
+    public void enableStoredLayers(boolean enable) {
```
Comment (UniQP) [inline main/src/cgeo/geocaching/maps/mapsforge/v6/caches/CachesBundle.java]: Checkstyle: Parameter enable should be final.

**Comment by Lineflyer** [comment]
@rsudev  Is this fine to `be merged from your side?

**Comment by pstorch** [comment]
retest this please

**Comment by rsudev** [comment]
I have to address the checkstyle issues still, hopefully can do this today evening

**Pull request merged**